README.md

```diff
@@ -28,7 +28,7 @@ Run the tool using `go run cmd/main.go`.
 
 The output ISO is written to `output/fleeting.iso`.
 
-Boot the ISO in a VM with at least 4096MiB of RAM. No storage is required.
+Boot the ISO in a VM with at least 8192MiB of RAM. No storage is required.
```

Comment:
Now that we are able to boot up the agents with ISO and we have this HW configuration https://github.com/openshift-agent-team/fleeting/pull/13/files#diff-514469c0981d360617b844887d7bed1da07f0dd12b8198001b83367b17319d4dR11, please update this information.

Reply:
Yeah, the whole README needs to be updated, but there's probably no point until we have a documented way for people to reproduce the environment.